src/hb-ot-font.cc

```diff
@@ -44,7 +44,7 @@
 struct hb_ot_face_metrics_accelerator_t
 {
   unsigned int num_metrics;
-  unsigned int num_advances;
+  uint32_t num_advances;
```

Comment:
needed as https://github.com/behdad/harfbuzz/blob/79e8e27ffd3da29ca27d3aebd2ef425bf1cb7f9d/src/hb-ot-font.cc#L141 glyph is uint32_t so this should be also (or I should cast there instead)